classes/Referrer.php

```diff
@@ -359,7 +359,7 @@ public static function refreshIndex($referrers = null)
      *
      * @param int $idReferrer Referrer ID
      * @param int $idProduct Product ID
-     * @param int $idEmployee Employee ID
+     * @param Employee|null $idEmployee Employee ID
      */
     public static function getAjaxProduct($idReferrer, $idProduct, $idEmployee = null)
```

Comment:
Variable name is confusing:
```suggestion
     * @param Employee|null $employee Employee
     */
    public static function getAjaxProduct($idReferrer, $idProduct, $employee = null)
```

Reply:
I disagree. The scope of this PR is validate PHPStan with a minimal of BC Breaks.

Reply:
Fixed